docs/sovereign/token-management.md

```diff
@@ -68,7 +68,7 @@
 
 - Register the sovereign token identifier on the main chain bridge contract:
     ```bash
-    registerToken
+    registerSovereignToken
```

Comment:
Can't we compress all these into a single script/call?

Reply:
I think this is ok because you need to register once and maybe deposit multiple times.